### PR TITLE
Add BERTClassifier

### PR DESCRIPTION
**Pull request by hkvision**

### CODE REVIEW

File: pyzoo/zoo/tfpark/text/bert.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            output_layer = tf.nn.dropout(output_layer, keep_prob=0.9)
+
+        logits = tf.matmul(output_layer, output_weights, transpose_b=True)
+        logits = tf.nn.bias_add(logits, output_bias)
+
+        tvars = tf.trainable_variables()
+        if params["init_checkpoint"]:
+            assignment_map, initialized_variable_names =\
+                modeling.get_assignment_map_from_checkpoint(tvars, params["init_checkpoint"])
+            tf.train.init_from_checkpoint(params["init_checkpoint"], assignment_map)
```
Comment (yangw1234) [inline pyzoo/zoo/tfpark/text/bert.py]: this does not work, estimator will call tf.global_variable_initializer after model creation

Comment (yangw1234) [reply pyzoo/zoo/tfpark/text/bert.py]: yeah it does work, my bad

**Comment by hkvision** [comment]
@jason-dai Take a look at the current structure?
If users want to customize their bert output, they can easily extend BERTBaseEstimator and create model_fn using _bert_model_fn, just like what BERTClassifier does.

**Comment by hkvision** [comment]
@jason-dai Any comments?

**Comment by hkvision** [comment]
@yangw1234 @qiuxin2012 Any comments?

File: pyzoo/zoo/examples/tensorflow/tfpark/run_classifier_bert.py [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+from optparse import OptionParser
+
+import tensorflow as tf
+from zoo.common.nncontext import *
+from zoo.pipeline.api.net import TFDataset
+from zoo.tfpark.text.estimator import BERTClassifier, bert_input_fn
+from bert import tokenization
+
+
+# Copy code from BERT run_classifier.py since import run_classifier will have error in imports such as
```
Comment (yangw1234) [inline pyzoo/zoo/examples/tensorflow/tfpark/run_classifier_bert.py]: should we preserve their license?

Comment (hkvision) [reply pyzoo/zoo/examples/tensorflow/tfpark/run_classifier_bert.py]: Oh, I will remove this example before merging this PR. Put here for review the usage of BERTClassifier purpose.

Comment (hkvision) [reply pyzoo/zoo/examples/tensorflow/tfpark/run_classifier_bert.py]: Removed this example.

**Comment by hkvision** [comment]
Jenkins passed: http://10.239.47.211:18888/job/ZOO-PR-Validation/2159/

**Comment by hkvision** [comment]
Merge it first. If you have further comments for modification, I will address them in another PR. @jason-dai @yangw1234

**Pull request merged**